<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="Gateway-PC" name="com.schemaxtech.testAutomationFrameworkTest.Plant2" tests="5" failures="0" timestamp="26 Dec 2019 10:08:59 GMT" time="3.062" errors="1">
  <testcase name="createPlant" time="0.094" classname="com.schemaxtech.testAutomationFrameworkTest.Plant2"/>
  <testcase name="getPlant" time="0.153" classname="com.schemaxtech.testAutomationFrameworkTest.Plant2"/>
  <testcase name="createPlant" time="2.702" classname="com.schemaxtech.testAutomationFrameworkTest.Plant2">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.schemaxtech.testAutomationFramework.TestAutomationUtil.verifyResponse(TestAutomationUtil.java:323)
at com.schemaxtech.testAutomationFrameworkTest.Plant2.createPlant(Plant2.java:89)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createPlant -->
  <testcase name="getPlant" time="0.113" classname="com.schemaxtech.testAutomationFrameworkTest.Plant2"/>
  <testcase name="updatePlant" time="0.000" classname="com.schemaxtech.testAutomationFrameworkTest.Plant2">
    <skipped/>
  </testcase> <!-- updatePlant -->
</testsuite> <!-- com.schemaxtech.testAutomationFrameworkTest.Plant2 -->
